.github/workflows/format-fix.yaml

```diff
@@ -0,0 +1,75 @@
+name: Phlex fix format
+run-name: ${{ github.actor }} fixing code format
+
+on:
+  issue_comment:
+    types:
+      - created
+
+permissions:
+   pull-requests: write
+   contents: write
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    name: Check for phlexbot token
+    if: ${{ github.event.issue.pull_request }}
+
+    outputs:
+      match_result: ${{ steps.check_comment.outputs.match_result }}
+      ref_name: ${{ steps.check_comment.outputs.ref_name }}
+
+    steps:
+      - id: check_comment
+        name: Check comment
+        run: |
+          comment="${{ github.event.comment.body }}"
+          match_result=$(echo $comment | grep -qE '@phlexbot[[:space:]]+format' && echo match || echo non_match)
+          echo match_result="$match_result" >> $GITHUB_OUTPUT
+
+          ISSUE_NUMBER=${{ github.event.issue.number }}
+          REPO="${{ github.repository }}"
+          gh api "repos/$REPO/pulls/$ISSUE_NUMBER" > pr.json
+          echo "ref_name=$(jq -r .head.ref pr.json)" >> $GITHUB_OUTPUT
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+  apply_formatting:
+    runs-on: ubuntu-latest
+    name: Apply formatting
+    needs: check
+    if: ${{ needs.check.outputs.match_result == 'match' }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: phlex-src
+          ref: ${{ needs.check.outputs.ref_name }}
+
+      - uses: DoozyX/clang-format-lint-action@v0.20
+        with:
+          source: "./phlex-src"
+          clangFormatVersion: 20
+          inplace: True
+
+      - uses: EndBug/add-and-commit@v9
+        id: add_and_commit
+        with:
+          cwd: phlex-src
+          author_name: "github-actions[bot]"
+          author_email: "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Should this be generated somehow or filled in via `${{ ... }}`?

Reply:
Not necessary, I just worried it was somehow meaningful to GitHub in a way that affected record keeping or traceability.